__tests__/app.test.js

```diff
@@ -380,4 +380,27 @@ describe("DELETE /api/comments/:comment_id",()=>{
       expect(msg).toBe("Bad Request")
     })
   })
-})
\ No newline at end of file
+})
+
+describe("GET /api/users",()=>{
+  test("200: Responds with an array of user objects",()=>{
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body:{users}})=>{
+      expect(Array.isArray(users)).toBe(true)
+      expect(users.length).toBe(4)
+      users.forEach(user=>{
+        expect(user).toMatchObject({
+          username:expect.any(String),
+          name:expect.any(String),
+          avatar_url:expect.any(String)
+
+        })
+      })
+    })
+  })
```

Comment:
Really nice test 🥇 I always like testing a GET request - once you're set up you stop having to worry about errors so much!